tasks/create-advisory-internal-request/README.md

```diff
@@ -9,12 +9,17 @@ the ReleasePlanAdmission and Application from the Snapshot are also used. The ad
 |--------------------------|-------------------------------------------------------------------------------------------|----------|-----------------------------|
 | jsonKey                  | The json key containing the advisory data                                                 | Yes      | .releaseNotes               |
 | releasePlanAdmissionPath | Path to the JSON file of the ReleasePlanAdmission in the data workspace                   | Yes      | release_plan_admission.json |
+| releaseServiceConfigPath | Path to the JSON file of the ReleaseServiceConfig in the data workspace                   | No       |                             |
 | snapshotPath             | Path to the JSON file of the Snapshot spec in the data workspace                          | Yes      | snapshot_spec.json          |
 | dataPath                 | Path to data JSON in the data workspace                                                   | Yes      | data.json                   |
 | request                  | Type of request to be created                                                             | Yes      | create-advisory             |
 | synchronously            | Whether the task should wait for InternalRequests to complete                             | Yes      | true                        |
 | pipelineRunUid           | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                             |
 
+## Changes in 1.3.0
```

Comment:
It's a breaking change, so should be 2.0.0?

Reply:
Good call, fixed